docs/update.md

```diff
@@ -95,40 +95,53 @@ You can upgrade the Operator and CRD as follows, considering the Operator uses
 ### Upgrade via Helm
 
 If you have [installed the Operator using Helm](helm.md), you can upgrade the
-Operator with the `helm upgrade` command.
+Operator deployment with the `helm upgrade` command.
 
-!!! note
-
-    You can use `helm upgrade` to upgrade the Operator. But the database management system (Percona Distribution for PostgreSQL) should be upgraded in the same way whether you used Helm to install it or not.
+The `helm upgrade` command updates only the Operator deployment. The update flow  for the database management system (Percona Distribution for PostgreSQL) is the same for all installation methods, whether it was installed via Helm or `kubectl`.
 
-1. Update the [Custom Resource Definition :octicons-link-external-16:](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
-    for the Operator, taking it from the official repository on Github, and do
-    the same for the Role-based access control:
+1. You must have the compatible version of the Custom Resource Definition. Starting with version 2.7.0, you can check it using the following command:
 
     ``` {.bash data-prompt="$" }
-    $ kubectl apply --server-side --force-conflicts -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/crd.yaml
-    $ kubectl apply -f https://raw.githubusercontent.com/percona/percona-postgresql-operator/v{{ release }}/deploy/rbac.yaml -n postgres-operator
+    $ kubectl get crd perconapgclusters.pgv2.percona.com --show-labels
     ```
-
-2. If you installed the Operator with no [customized parameters :octicons-link-external-16:](https://github.com/percona/percona-helm-charts/tree/main/charts/pg-operator#installing-the-chart), the upgrade can be done as follows: 
+    
+2. Update the [Custom Resource Definition :octicons-link-external-16:](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
```

Comment:
I'm not sure if that's the case, but maybe we need to mention the CRD version shouldn't be downgraded (to avoid conflicts with versions installed on other namespaces)